packages/orchestration/src/orchestration-api.ts

```diff
@@ -104,7 +104,12 @@ export interface BaseChainInfo<N extends KnownNamespace = KnownNamespace> {
   cctpDestinationDomain?: number;
 }
 
-export type ChainInfo = Readonly<BaseChainInfo | CosmosChainInfo>;
+/**
+ * Shape that `ChainHub` is expecting
+ */
+export type ChainInfo = Readonly<
+  (BaseChainInfo & { chainId: string }) | CosmosChainInfo
```

Comment:
when `chainId` is not part of `CosmosChainInfo`, what is its domain? I'm pretty sure it's `${namespace}:${reference}` but not certain. Why do we need it when it's redundant with those? Regardless this should explain how it differs from `CosmosChainInfo['chainId']`

Reply:
Please try dropping `chainId` from the non-Cosmos case because it has a different meaning.

Reply:
Understood. It was easier to layer this change on top versus figuring out the proper place(s) to rebase. Please see: https://github.com/Agoric/agoric-sdk/pull/11131/commits/c276d6b018b8e4322b72753c80bf483c418c32bb